tests/functional/spec/theme_nhse.spec.js

```diff
@@ -0,0 +1,16 @@
+import RadioPage from "../generated_pages/theme_ons_nhs/radio.page";
+import { expect } from "@wdio/globals";
+
+describe("Theme NHSE", () => {
```

Comment:
Is NHSE the name of the theme? Shouldn't it be ONS-NHS?

Reply:
Feels like a typo to me but yeah can see its down as NHSE in multiple cases on the card